tests/users/test_queries.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import uuid
+from graphql_relay import to_global_id
+
+from django.contrib.auth import get_user_model
+from creator.organizations.factories import OrganizationFactory
+from creator.organizations.models import Organization
+from creator.users.factories import UserFactory
+
+User = get_user_model()
+
+ALL_USERS_ORGANIZATION = """
+query ($organizations: [ID]) {
+  allUsers (organizations: $organizations){
```

Comment:
just a note - allStudies only allows a single organization id. Allowing a list is probably more flexible, but we should make sure to align the other endpoints to match.

Reply:
honestly, i wasn't able to figure out how to do it in the case of a single org id, because it's a many-to-many relationship. i agree, i can add that functionality to `allStudies`

Reply:
I'd agree with being consistent with other queries. Allowing to filter by multiple orgs would be nice, but we don't have that need right now and it's not how we do things elsewhere. You can probably achieve this pretty easily by swapping out to `GlobalIDFilter` instead of the multiple choice.